docs/spec/staking/end_block.md

```diff
@@ -16,3 +16,34 @@ EndBlock() ValidatorSetChanges
     ClearTendermintUpdates()
     return vsc
 ```
+
+
+## CompleteUnbonding
+
+Complete the unbonding and transfer the coins to the delegate. Perform any
+slashing that occurred during the unbonding period.
+
+```golang
+unbondingQueue(currTime time.Time):
+    for all unbondings whose CompleteTime < currTime:
```

Comment:
Although order shouldn't matter, I think we should specify the order of iteration here (in order of oldest to newest presumably).